apps/01PigLatin.js

```diff
@@ -7,11 +7,87 @@ prompt.start();
 
 function pigLatin(word) {
 
-    // Your code here
+  // Concept Check
+  //
+  // Please explain why we are using String.prototype.toLowerCase() here.
+  // For more info:
+  // https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/String/toLowerCase
+  //
+  // Your Answer:
+  //To convert all strings to lower case letters for comparison against our strings.
+  //An input with upper case letters would not return true to match our strings.
+  word  = word.toLowerCase();
 
+  // Concept Check
+  //
+  // What is the significance of setting vowelIndex to -1?
+  // This might help:
+  // https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/String/lastIndexOf
+  //
+  // Your Answer:
+  //vowelIndex is set to -1 because the index of the first letter of the String
+  //is 0.  We want to evaluate the first position for a vowel as well.  If it
+  //were set to 0, there could be a vowel in the first position of the String
+  //but it would not be checked.
```

Comment:
Good start. Also [String.prototype.indexOf()](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/String/indexOf) returns -1 if it can't find something.